srcpkgs/osg/template

```diff
@@ -23,6 +23,12 @@ homepage="http://www.openscenegraph.org"
 distfiles="https://github.com/openscenegraph/OpenSceneGraph/archive/OpenSceneGraph-${version}.tar.gz"
 checksum=930eb46f05781a76883ec16c5f49cfb29a059421db131005d75bec4d78401fd5
 
+# necessary because OSG's cmake script don't properly account for RelWithDebInfo
+# this results in shared libraries with the 'rd' prefix added, but the cmake
+# library find scripts for OSG only assume either release or debug, so these
+# libs will never be found
+nodebug=1
```

Comment:
I dislike completely disabling debug symbols for this. I will probably revert the build style change and we should just set `RelWithDebInfo` only for packages that just ignore our cflags.

Reply:
tbh, i wonder which one would require more manual intervention... another option would be patching osg to recognize `RelWithDebInfo`, which I looked into but their cmake macros seemed like too much of a mess to make it worth it.

Reply:
Afaik the amount of packages that require `RelWithDebInfo` is actually quite small.

Reply:
OKay then, sounds good